src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -96,7 +98,13 @@ ASR::Module_t* load_module(Allocator &al, SymbolTable *symtab,
     Result<std::string> rinfile = get_full_path(infile0, rl_path, ltypes,
         numpy);
     if (!rinfile.ok) {
-        err("Could not find the module '" + infile0 + "'", loc);
+        if (getenv("LPYTHON_PY_MOD_PATH") != NULL) {
+            rinfile = get_full_path(infile0, getenv("LPYTHON_PY_MOD_PATH"), ltypes,
+                numpy);
+        }
```

Comment:
The `LPYTHON_PY_MOD_PATH` is only used for CPython tests, and thus should not be needed here, as LPython does not use this environment variable.

Reply:
Then, how should we find the modules that are present in the current directory of tests? CPython does that itself.

Reply:
Please open up an issue for this --- we have to find all imported modules relative to the file they are imported from, not relative to the current directory.